src/util/math.ts

```diff
@@ -14,7 +14,12 @@ export function accuratePow64(x: f64, y: f64): f64 {
     // This speculative optimization leads to loose precisions like 10 ** 208 != 1e208
     // or/and 10 ** -5 != 1e-5 anymore. For avoid this behaviour we are forcing exponent
     // to fractional form and compensate this afterwards.
-    if (isFinite(y) && Math.abs(y) >= 2 && Math.trunc(y) == y) {
+    if (
+      isFinite(y) &&
+      !isPowerOf2(x) &&
```

Comment:
`isPowerOf2(Number.MAX_SAFE_INTEGER)` seems to return a false-positive.

Reply:
Never mind, I'm stupid.

Reply:
Oh, but `isPowerOf2(Number.MAX_SAFE_INTEGER + 3)` returns `true`.

Reply:
I guess for so huge arguments it doesn't really mater because everything > `Number.MAX_SAFE_INTEGER` doesn't have fractional part at all.

Reply:
Ah okay. That's good to know.